docs/helm-chart-checks.md

```diff
@@ -124,59 +124,77 @@ This section provides help on the basic usage of Helm chart checks with the podm
 - Run a subset of the checks:
 
   ```
-  $ podman run --rm -i \
-          -e KUBECONFIG=/.kube/config \
-          -v "${HOME}/.kube":/.kube \
+  $ podman run --rm -i                                  \
+          -e KUBECONFIG=/.kube/config                   \
+          -v "${HOME}/.kube":/.kube                     \
```

Comment:
Is this your IDE doing this why the extra spaces?

Reply:
similar to https://github.com/redhat-certification/chart-verifier/pull/225, for readability.